partner_design_mockup/static/src/xml/partner_search.xml

```diff
@@ -0,0 +1,27 @@
+<templates>
+    <t t-extend="ControlPanel">
+        <t t-jquery="ol.breadcrumb" t-operation="attributes">
+            <attribute name="style">width:20%</attribute>
+        </t>
+        <t t-jquery="ol.breadcrumb" t-operation="after">
+            <div class="search_place_view">
+                <select id="search_place" name="search_place" class="search_place_input">
+                    <option value="at_contact">vid kontakt</option>
+                    <option value="based_on_authority_needs">Utifran Myndighetens behov</option>
```

Comment:
Utifran -> Utifrån (If we allow swedish letters here)

Reply:
Hi @nils-vertel I updated now.